StreamChatUITestsAppUITests/General.swift

```diff
@@ -0,0 +1,14 @@
+//
+// Copyright © 2022 Stream.io Inc. All rights reserved.
+//
+
+@_exported import StreamChatTestHelpers
```

Comment:
@martinmitrevski this notation works similarly to precompiled headers in Objective-C. 
It's used widely & also suggested by Swift team.
This definition makes the test helpers available across the whole target. Teams have never run into brittle experience with this, rather it helped them to leverage our tools.

One of possible cases this catches is eg helper that asserts against the equality `XCTAssertEqual`, which is commonly used by every single developer.
It's really easy to forget to import the test helpers and thus leverage the basic assert instead of the improved one, that leverages Difference under the hood and gives us better contextual info when test fails.

The above notation covers these cases in our test targets with ease, simply because there's no room for mistake.

Reply:
thanks for the explanation, @bielikb! Yesterday I've done some reading on @_exported and found this discussion: https://forums.swift.org/t/exported-and-fixing-import-visibility/9415/58. The part that worried me was this:
"@_exported is an implementation detail (as indicated by the leading underscore); its semantics are whatever it does, and can be changed at any point, so you should not depend on it doing anything specific."

Reply:
I see, 


I like to follow 80/20 rule :)
since this is non-production code, we might want to balance the above fact with the ease of use/adoption of the test helpers in the current and future tests that we will write.

The use of `@_exported` in our own defined test target is a rather safe, the compiler directive is there for couple of years already. Any related deprecations/issues with this approach will lead to the linking failure of our XCTestHelpers.
I can add this point to our docs. 
Thanks!